code/_onclick/item_attack.dm

```diff
@@ -25,6 +25,13 @@
 	I.attack(src, user)
 
 /mob/living/proc/attacked_by(obj/item/I, mob/living/user, def_zone)
+
+	if(isdrone(user))
```

Comment:
This check is pretty snowflakey here,it would be better if it was on the drone's side,somehow